mmcls/models/losses/cross_entropy_loss.py

```diff
@@ -178,12 +191,19 @@ def forward(self,
         else:
             class_weight = None
 
+        # only BCE loss has pos_weight
+        if self.pos_weight is not None and self.use_sigmoid:
+            pos_weight = cls_score.new_tensor(self.pos_weight)
```

Comment:
Since `pos_weight` is only used in BCE loss, just pass the parameter here, and other cross entropy function won't need to add this useless argument.
```suggestion
            pos_weight = cls_score.new_tensor(self.pos_weight)
            kwargs.update({'pos_weight': pos_weight})
```

Reply:
fixed